cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -78,6 +78,20 @@ static const size_t egl_vendor_globs_len =
 // FIXME: this doesn't yet work with libGLX and libglvnd redirector
 // FIXME: this still doesn't work with the 361 driver
 static const char *nvidia_globs[] = {
+	"gbm/nvidia-drm_gbm.so*",
+	"libnvidia-allocator.so*",
+	"libnvidia-api.so*",
+	"libnvidia-cbl.so*",
+	"libnvidia-cfg.so*",
+	"libnvidia-compiler-next.so*",
+	"libnvidia-egl-gbm.so*",
+	"libnvidia-ngx.so*",
+	"libnvidia-nscq.so*",
+	"libnvidia-nvvm.so*",
+	"libnvidia-pkcs11-openssl3.so*",
```

Comment:
I'd add a comment with a TODO to revisit if we need to expose these libs

Reply:
I think we should not do that. My stance is that we just don't want to be in the business of understanding binary nvidia stack. We are asked to provide it and so we do. The only distiction we should care about is if it's a part of the runtime or if it's a part of the platform (e.g. x11 server, service, controller code, etc).